### PR TITLE
Setup-hello-microverse

### PR DESCRIPTION
**Pull request by hasenat-eqbali**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+- LinkedIn: [Hasenat Eqbali](https://linkedin.com/in/hasenat-eqbali)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is MIT(./MIT.md) licensed.
+
```
Comment (ha-manel) [inline README.md]: - [ ] Kindly add the missing relevant sections of the README file (setup...) 👍 
- [ ] Kindly make the MIT text in the license section a link that takes to the actual LICENSE (which you already have in your project)

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```
Comment (ha-manel) [inline test.md:1]: - [ ] This file should be ignored which means when you push to the remote branch, it's not pushed with the rest of the files. I can see that you added it to `.gitignore` so it means you added it after you commited and pushed it to the remote branch. Please refer to [this](https://dev.to/morinoko/useful-git-commands-for-removing-accidentally-pushed-or-committed-files-2ld#:~:text=Remove%20a%20file%20you%20already%20pushed%20to%20the%20git%20repository&text=To%20prevent%20it%20from%20being,from%20your%20system%20as%20well.) to fix the issue 👍

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @hasenat-eqbali 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Github flow is used correctly 👍 

## Required Changes:

- [ ] Kindly update the pull request description and title, they should be descriptive enough of the changes that were made in the pull request 👍 
- [ ] From now on, kindly choose the branch's name more carefully. It should describe an action of what to be done in the branch. Example: "setup-hello-microverse" 💯 
- [ ] Kindly try to follow best practices when writing commit messages. They should be capitalized sentences in the imperative form. Example: "Add linters" instead of "linters added" 👍 

_Kindly check the comments below_


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by hasenat-eqbali** [comment]
Hi

Thanks for your comments. I did not understand what I should do in
README.md. Also, I removed test.md with the help of git rm --cached
name_of_file  but it is still in the branch.

Thanks
Hasenat

**Pull request closed without merging**